dotCMS/src/main/java/com/dotmarketing/util/ImportUtil.java

```diff
@@ -1096,9 +1096,14 @@ private static void importLine ( String[] line, String currentHostId, Structure
             }
 
             for (Contentlet cont : contentlets) {
+
+                //Clean up any existing workflow action
+                cont.setStringProperty(Contentlet.WORKFLOW_ACTION_KEY, null);
```

Comment:
I would say to encapsulate this kind of things on methods, instead of using constants.